Modules/cmathmodule.c

```diff
@@ -1274,23 +1226,22 @@ cmath_exec(PyObject *mod)
     if (PyModule_AddObject(mod, "tau", PyFloat_FromDouble(Py_MATH_TAU)) < 0) {
         return -1;
     }
-    if (PyModule_AddObject(mod, "inf", PyFloat_FromDouble(m_inf())) < 0) {
+    if (PyModule_AddObject(mod, "inf", PyFloat_FromDouble(Py_HUGE_VAL)) < 0) {
```

Comment:
Should probably use `Py_INFINITY` here, since we're changing this line anyway. (I don't want to clutter this PR by changing _all_ instances of `Py_HUGE_VAL` to `Py_INFINITY`; that can happen in a separate cleanup PR.)